tests/internal/testutils/templates.go

```diff
@@ -32,7 +32,7 @@ func FindAllTemplates(t *testing.T, templateUrl string) []TemplateInfo {
 		templateName := t.Name
 		templatePath := templateName
 		if templateUrl != "" {
-			templatePath = path.Join(templateUrl, templateName)
+			templatePath = filepath.Join(templateUrl, templateName)
```

Comment:
Made this change after noticing the paths on Windows contained a mixture of backward and forward slashes. Using `filepath` instead of `path` fixes this up.